db/migrate/20230209163834_add_devise_to_users.rb

```diff
@@ -0,0 +1,54 @@
+# frozen_string_literal: true
+
+class AddDeviseToUsers < ActiveRecord::Migration[7.0]
+  def self.up
+    change_table :users do |t|
+      ## Database authenticatable
+      t.string :email,              null: false, default: ""
+      t.string :encrypted_password, null: false, default: ""
+
+      ## Recoverable      t.string   :confirmation_token
+      t.datetime :confirmed_at
+      t.datetime :confirmation_sent_at
+      t.string   :unconfirmed_email
+      t.string   :reset_password_token
+      t.datetime :reset_password_sent_at
+
+      ## Rememberable
+      t.datetime :remember_created_at
+
+      ## Trackable
+      # t.integer  :sign_in_count, default: 0, null: false
+      # t.datetime :current_sign_in_at
+      # t.datetime :last_sign_in_at
+      # t.string   :current_sign_in_ip
+      # t.string   :last_sign_in_ip
+
+      ## Confirmable
+      t.string   :confirmation_token
+      t.datetime :confirmed_at
```

Comment:
- [ ] Nice work done a your migration file, kindly remove `:confirmed_at, :confirmation_sent_at, :unconfirmed_email` because they have already been declared above and its causing the app to throw the below error in the screenshot during migration.

![migrate_wahala](https://user-images.githubusercontent.com/48371156/217951276-db602965-f111-48e9-8e49-d68afde3c7d1.png)